src/app/constants.js

```diff
@@ -0,0 +1,15 @@
+// Note - this file should only include standard constants,
+// agreed to by UX and Engineering
+
+// Colours
+export const C_EBON = '#222222';
+export const C_POSTBOX = '#BB1919';
+export const C_STORM = '#404040';
+export const C_WHITE = '#FFFFFF';
+
+// Font family
+export const FF_NEWS = 'ReithSans, Arial, Helvetica, freesans, sans-serif';
```

Comment:
I would also suggest we handle font size and line height in the same way as this because eventually we'll need the ability to override font-family, font-size and line-height for the WS sites.

Reply:
Agreed. I'm not sure whether we have a particular range of font sizes from UX. I presume we should just follow what are provided in GEL Typography, for Latin-based services, at least. 
https://github.com/bbc/gel-typography/blob/master/lib/_settings.scss

We'll need to discuss how we can handle typography classes in this Styled Components system.

Reply:
How many colors are we intending to have? Sorting or grouping them might be necessary if the number grows.

Reply:
@DemonusPC Our colours are here, in a Sass file that is used for other projects: https://github.com/bbc/gs-sass-tools/blob/master/settings/_news-colours.scss#L31-L47 we might not be using all of these colours in this project. I think it makes sense to add them to this file when they are first used.

Reply:
Yea that makes sense now, I was thinking about sorting them via Hue but this is good too.